### PR TITLE
[hap2][document] Create how to use livestatus module

### PR DESCRIPTION
**Pull request by Mnakagawa**
Related: #1948

### CODE REVIEW

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# How to setup Livestatus API
+-----------------------------------------------------------------------------------
+
+You should install livestatus module, if you use hap2_nagios_livestatus.py plugin.
+
+This document describe for how to install livestatus module on CentOS 7 and Ubuntu 14.04 LTS.
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: - describe -> describes
- for: unnecessary (should be removed)

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# How to setup Livestatus API
+-----------------------------------------------------------------------------------
+
+You should install livestatus module, if you use hap2_nagios_livestatus.py plugin.
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: , if -> if (comma is unnecessary)

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    # yum install epel-release
+    # yum install check-mk-livestatus 
+
+### Ubuntu 14.04 LTS
+
+    # apt-get install check-mk-livestatus
+
+## (2) Check livestauts library path
+
+You execute the following command on each OS to check the path of "livestatus.o".
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: - You execute -> Execute
- check -> see

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+### Ubuntu
+
+    # dpkg -L check-mk-livestatus
+
+## (3) Edit /etc/nagios/nagios.cfg to use an livestatus broker module
+
+Add the following line to /etc/nagios/nagios.cfg.
+
+Livestatus module make AF UNIX socket file.
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: make -> makes
or
creates

And in this context 'file' is countable.
AF UNIX socket file ->
an AF UNIX socket file

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+### Ubuntu 14.04 LTS
+
+    # apt-get install check-mk-livestatus
+
+## (2) Check livestauts library path
+
+You execute the following command on each OS to check the path of "livestatus.o".
+
+The path is replaced the next section's LIVESTATUS_LIB_PATH.
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: I cannot find 'LIVESTATUS_LIB_PATH' below.
You mean it's LIVESTATUS_LIB_FILE_PATH ?
If so,  I suggest the following sentence.

    The path obtained above should be LIVESTATUS_LIB_FILE_PATH in the next section.

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+```
+broker_module=LIVESTATUS_LIB_FILE_PATH config_file=ANY_SOCKET_FILE_PATH
+```
+
+## (3') Setup xinetd
+
+If you want to activate hap2_nagios_livestatus.py outside the nagios local machine, 
+
+you should setup the xinetd. This section describe how to setup xinetd.
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: describe -> describes

(if desired)
how to setup xinetd -> how to do it

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+### Installation to CentOS 7
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: to -> on ?

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+    # yum install xinetd
+
+### Installation to Ubuntu 14.04 LTS
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: to -> on ?

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+    # service xinetd start
+
+
+## (4) Start nagios
+
+    # service nagios start
+
+## (4') Test
+
+You can test whether livestatus module is operating correctly by the following command.
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: (if desired)
test ->
  see
  confirm

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+
+    # echo 'GET contacts' | unixcat ANY_FILE_PATH
+
+If you confirm as the following output, livestatus module setting is completed.
```
Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: Where is 'the following output' ?

Comment (kz0817) [inline doc/server/hap2/how-to-setup-livestatus.md]: confirm is transitive verb. an object is necessary.

Instead, for example,
- If the following output is shown
- if you see the following output

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 11]
```diff
@@ -0,0 +1,105 @@
+# How to setup Livestatus API
+-----------------------------------------------------------------------------------
+
+You should install livestatus module, if you use hap2_nagios_livestatus.py plugin.
+
+This document describe for how to install livestatus module on CentOS 7 and Ubuntu 14.04 LTS.
+
+This document is written with the assumption that Nagios is installed.
```
Comment (cosmo0920) [inline doc/server/hap2/how-to-setup-livestatus.md]: This document assumes that Nagios is installed.

英語では生物でない語も主語になることができます。
この用法は例えば、The newspaper says that .... 
のように「新聞は〜と報道している」を英語らしい表現で書いた文に見られます。

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+
+    # yum install xinetd
+
+### Installation to Ubuntu 14.04 LTS
+
+    # apt-get install xinetd
+
+### Put livestatus configuration file
+
+The following is an example configuration file.
```
Comment (cosmo0920) [inline doc/server/hap2/how-to-setup-livestatus.md]: An example configuration file is as follows:

```
service livestatus
{
<snip>
}
```

Put it to `/etc/xinetd.d/livstatus`.

という構成の方がいいかも。

File: doc/server/hap2/how-to-setup-livestatus.md [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+	flags       = NODELAY
+	user        = nagios
+	server      = /usr/bin/unixcat
+	server_args = ANY_SOCKET_FILE_PATH
+	disable     = no
+#	only_from   = 127.0.0.1
+}
+```
+
+### xinetd start
```
Comment (cosmo0920) [inline doc/server/hap2/how-to-setup-livestatus.md]: Start xinetd

File: doc/server/hap2/how-to-setup-livestatus.md (hunk 10 above)
Comment (Mnakagawa) [inline doc/server/hap2/how-to-setup-livestatus.md]: > Where is 'the following output' ?

Oh...
I forgot to show. Thank you.

**Comment by Mnakagawa** [comment]
I fixed your indicated matter!
Thank you.

**Comment by cosmo0920** [comment]
> I fixed your indicated matter!

Oh, this PR contains "one" matter?
It sounds you are great English writer!

Just joking around, haha. :-P

**Comment by Mnakagawa** [comment]
:facepunch:

**Comment by Mnakagawa** [comment]
Could I merge this PR?

**Comment by cosmo0920** [comment]
LGTM. :shipit: !

**Comment by Mnakagawa** [comment]
Thanks:white_flower:

**Pull request merged**